### PR TITLE
Added EAD3 column to crosswalk.

### PR DESCRIPTION
**Pull request by ruthtillman**
Added draft column of EAD3 elements to the table of DACS to EAD and MARC (now DACS to EAD 2002, EAD3, and MARC) table.

### CODE REVIEW

**Comment by milesnimer** [comment]
While I agree that there is a need for this addition to the crosswalk, there are a couple of things that I felt might need some adjustment before this is added to the crosswalk. First, it seems like the current version of the proposal has purposely excluded mention of <relations> and its sub-element entirely. While <relations> is considered to be an experimental element, it is valid in EAD3 and its exclusion would seem to be a tacit recommendation against its use. For this reason I would recommend that the <relation> element be added to all applicable elements in the crosswalk. This would mean edits to the file in lines 798 (&lt;origination&gt; or &lt;relation&gt;), 858 (&lt;recordid&gt; instanceurl attribute, &lt;representation&gt;, &lt;otherfindaid&gt;, or &lt;relation&gt;), 894 (&lt;originalsloc&gt; or &lt;relation&gt;), 900 (&lt;altformatavail&gt; or &lt;relation&gt;), 906 (&lt;relatedmaterial&gt; or &lt;relation&gt;), and 912 (&lt;bibliography&gt; or &lt;relation&gt;). 
 
In the case of the 4.6 Finding Aids element mapping, I also wondered whether the <recordid> instanceurl attribute and <representation> mappings were necessary. It had always seemed to me that the benefit of this element was to identify other tools, beyond the current finding aid, that might be of use to researchers. Both of these elements should be providing pointers to the finding aid that the researcher is already accessing (either the original XML file, or to derivative copies with equivalent content). As such, they don't seem necessary as part of this crosswalk.

**Comment by ruthtillman** [comment]
Been waiting to see if there were other comments on this, but I can explain my thinking. I see `<relation>` as an AND POSSIBLY ALSO rather than an OR. I can give an example with "origination." Since DACS is a content vs. an encoding standard, one might say it just matters that as there's some kind of origination metadata... but `<relation>` on its own doesn't have a way of saying "hey this is a relation to the originator and thus fulfills the origination requirement." That both means there's no good way to validate encoding and there's no way other than reading the contents of the `<descriptivenote>` to ensure any visual kind of validation the way the inclusion of an `<origination>` element does. Therefore I don't know if I can recommend `<origination>` OR `<relation>` because the latter can't specify what parts of DACS it's fulfilling.

I have a couple ideas about how this could be included. The easiest would be a footnote attaching to all elements which could conceivably be augmented by `<relation>` and spelling out in the footnote that this element is a way to augment this content requirement.

And I'd like to hear other perspectives, maybe I'm thinking too much in terms of encoding and `<relation>` really does meet the need for these cases with no `<origination>` etc.

**Comment by helrond** [comment]
@ruthtillman's explanation makes sense to my admittedly non-EAD3-expert mind. I like the footnote idea as a middle ground between omitting `<relation>` completely and making awkward/inaccurate equivalencies.

**Comment by milesnimer** [comment]
I agree with @ruthtillman that there are some challenges in making a clean, clear entry for the <relation> tag without getting perhaps overly proscriptive in the mapping recommendations. In order to have valid, clear tagging to adequately replace a straightforward `<origination>` element you need to specify both the relationtype and linkrole attributes, the latter of which requires a URI value to note the creator relationship (e.g., `<relation relationtype="cpfrelation" linkrole="http://purl.org/dc/elements/1.1/creator">`). Of course, this linkrole value could be taken from Dublin Core, or the Open Metadata Registry, or a locally defined ontology. Definitely messy--but on an element level it would seem that the mapping is still just using `<relation>`.

**Comment by ruthtillman** [comment]
Hmm, I like that for the most part (concern below), although I'd use @arcrole, which the TL defines as "A URI that describes the nature of the source of a link as relative to the target of the link" vs. @linkrole, "A URI that characterizes the nature of the remote resource to which a linking element refers" (in examples, arcrole = dc:creator, linkrole = foaf:person). If we were to follow through on that, my suggestion would be one of the following:

[2] The element `<relation>` offers the opportunity to either enhance or substitute for this element by providing a link, citation, or embedded XML which provides the information (e.g. an EAC-CPF file for the originator of the materials). Use the attribute `arcrole` on `<relation>` to include a URI specifying the role this related resource plays in the finding aid, e.g. `<relation relationtype="cpfrelation" arcrole="http://purl.org/dc/elements/1.1/creator" ...>`

Or, including "or `<relation>`[2]" within the table element, where the footnote remains essentially the same.

However, I have one concern: Links aren't a necessary part of <relation>, which can as easily include a citation, etc. @arcrole's definition expects a link (does it necessitate one?). Do we want to recommend the use of `<relation>` entirely for linked materials? That would seem unduly limiting but then we're back to "I don't know where I'd specify this."

**Comment by rockivist** [comment]
Hi all! Your friendly former TS-EAD co-chair here. I'm happy to see support for addressing `<relations>` in the DACS > EAD crosswalk. Please don't be scared off by the "experiment" status. We need people to use it so that we can make improvements from experience. This is particularly important when discussion areas where the relations model may duplicate the semantics of established elements link `<origination>`. As a community we need to acknowledge this and collectively weigh the benefits of one approach over the other and make a decision re: doing one or the other or both. Elements that might be modeled out in `<relations>` include origination, repository, relatedmaterial, separtedmaterial, otherfindaid, altformavail, acqinfo, custodhist, bibliography, and controlaccess to name those that come to mind immediately. 

I also want to chime in to second @ruthtillman and her assessment of `@arcrole` vs. `@linkrole`. `@arcrole` explicitly characterizes the nature of the link - or arc - between the materials being described and the related entity. So if you are providing a URI to identify the relationship as that of creator, or subject, or what have you, it goes there. 

The `@linkrole` attribute describes the nature of the linked resource itself. If you are linking to a document it might be a MIME type like application/pdf, or you could described the nature of the related entity with foaf as Ruth suggested. It's a little vague, and frankly much less important since the linked thing will likely self-identify in some way. If you are asking me I'd provide some guidance for `@arcrole` and ignore `@linkrole`.

**Comment by ruthtillman** [comment]
@rockivist Thanks! So we could provide correct guidance on `@arcrole`, do you see it as something to be used even when there's just a citation or only when there's a corresponding link to the related material?

**Comment by rockivist** [comment]
@ruthtillman I think it's a critical part of the `<relation>` data model. So yes, recommend including it whether the intellectual relation is accompanied by an actionable link or not.

TL;DR version: In EAC-CPF, there are three different relation elements: `<cpfRelation>`, `<functionRelation>`, and `<resourceRelation>`, each of which has a its own typing attribute, i.e. `@cpfRelationType`, `@functionRelationType`, and `@resourceRelationType`. Each also has `@xlink:arcrole`.  With EAD3 we decided to simplify that model down to a single `<relation>` element for a few reasons. 1) It allowed for the flexibility to allow different types of relations. 2) Experience with EAC-CPF showed that the closed value lists for the relation type attributes were useless bordering on problematic and in need of updating. Anticipating the RIC vocabs, we didn't want to perpetuate the EAC-CPF relation type vocabs, nor did we want to write our own. So we eliminated those relation types, pushing the semantics to `@arcrole`. This keeps EAD neutral in terms of not imposing a vocabulary for characterizing the nature of relationships, and also avoids needing to make regular updates to the standard as the vocab for such relations evolves. I expect EAC-CPF will shift to a model more closely resembling EAD3. All of which is to say that `@arcrole` is very important, and much more so in EAD3 than in EAC-CPF.

**Comment by milesnimer** [comment]
Thanks for the correction--you are right, I did mean `@arcrole`. Given the two options that @ruthtillman presents, I would still recommend including `<relation>` in the table itself, even if a footnote on the proper use of the `@arcrole` is needed for explanation.

**Comment by ruthtillman** [comment]
Tagging in @helrond and @meau just to see if this flies ok by you before i make the edit, since it'll affect this pull request...

For the following concepts, add `<relation>` as an "or": Origination, Finding Aids, Existence and Location of Originals, Existence and Location of Copies, Related Archival Materials, Publication Note (suggested by @milesnimer --is that you, Cory? Or just a surname similarity?) and possibly also Custodial History, Immediate Source of Acquisition, and Name and Location of Repository/Repository Identifier (as additionally suggested by @rockivist).

With a footnote to read along the lines of:

[2] The element `<relation>` offers the opportunity to either enhance or substitute for this element by providing a link, citation, or embedded XML which provides the information (e.g. an EAC-CPF file for the originator of the materials). Use the attribute arcrole on <relation> to include a URI specifying the role this related resource plays in the finding aid, e.g. `<relation relationtype="cpfrelation" arcrole="http://purl.org/dc/elements/1.1/creator" ...>`

**Comment by ruthtillman** [comment]
Updated per comments. I added guidance on `@arcrole` for origination, but did not feel comfortable laying out recommended arcroles for other fields, particularly as I worried they might be considered recommended practices vs. examples. I would recommend that there be some review and consensus if such recommendations are being made and that, in the interim, we go with the edits as is.
